ArgonBenchmarks/BenchmarkDotNet.Artifacts/ArgonBenchmarks.ArgonBenchmarks-20211107-204241.log

```diff
@@ -0,0 +1,1366 @@
+// Validating benchmarks:
```

Comment:
I made this comment yesterday but hadn't finished the review, heard you say yes to this when Steve asked so resolving.

Reply:
Or not, since the button is missing :/
![image](https://user-images.githubusercontent.com/68649564/141015758-38f64a39-c327-4238-9e8c-28201d015cb0.png)